cumulus_library/studies/core/reference_sql/builder_patient.sql

```diff
@@ -266,24 +266,33 @@ WITH temp_patient AS (
         patient AS p
     LEFT JOIN core__patient_ext_race AS er ON p.id = er.id
     LEFT JOIN core__patient_ext_ethnicity AS ee ON p.id = ee.id
+),
+
+-- Grab a single address to use for this patient, if present
+temp_address AS (
+    SELECT
+        tp.id,
+        
+        
+        
+        substr(max(t_address.addr_row.postalcode), 1, 3) AS postalcode3
+    FROM
```

Comment:
making a note to fix this the next time i'm opening a PR

Reply:
Ahem, that's my fault, sorry